cypher/cypher-docs/src/docs/dev/syntax/naming.asciidoc

```diff
@@ -5,22 +5,30 @@ We describe here rules and recommendations for the naming of node labels, relati
 
 == Naming rules
 
-* Must begin with an alphabetic letter.
+* Alphabetic letters:
+** Names should begin with an alphabetic letter.
 ** This includes "non-English" characters, such as `å`, `ä`, `ö`, `ü` etc.
-** If a leading non-alphabetic character is required, use backticks for escaping; e.g. `++`^n`++`.
-* Can contain numbers, but not as the first character.
+//** If a leading non-alphabetic character is required, use backticks for escaping; e.g. `++`^n`++`.
+* Numbers:
+** Names should not begin with a number without escaping.
```

Comment:
I would rather rewrite it as you suggested, first what is possible and then what should be avoided, and if you can't how to tackle it.

Reply:
For example, "Names can contain numbers but not as the first character, for example, `first1`.